atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -560,6 +561,17 @@ def get_build(self, metadata):
                 else:
                     extra['filesystem_koji_task_id'] = task_id
 
+        help_result = self.workflow.prebuild_results.get(AddHelpPlugin.key)
+        if help_result is not None:
+            try:
+                cgit_url = self.workflow.source.uri.replace('/rpms/', '/cgit/rpms/')
```

Comment:
How about formatting it as `{{git_url}}#{{commit}};{{filename}}`? Do we know if other koji plugins are using any scheme?

Sorry for discussing this after implementing, I expected the task to detect a git frontend to be much simplier